### PR TITLE
Chore - Update vfs and Pebble libs

### PR DESCRIPTION
**Pull request by coufalja**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -3,13 +3,46 @@ module github.com/lni/dragonboat/v3
...
 	github.com/VictoriaMetrics/metrics v1.18.1
 	github.com/cockroachdb/errors v1.9.0
-	github.com/cockroachdb/pebble v0.0.0-20220322140420-6e19e39957fb
+	github.com/cockroachdb/pebble v0.0.0-20220407171941-2120d145e292
 	github.com/golang/snappy v0.0.4
 	github.com/hashicorp/memberlist v0.3.1
 	github.com/juju/ratelimit v1.0.2-0.20191002062651-f60b32039441
 	github.com/lni/goutils v1.3.1-0.20220404072553-ddb2075d2587
-	github.com/lni/vfs v0.2.0
+	github.com/lni/vfs v0.2.1-0.20220407140350-6b33444f675b
```
Comment (coufalja) [inline go.mod]: Maybe tag a release?

File: internal/logdb/kv/pebble/kv_pebble.go [hunk 2]
```diff
@@ -374,5 +374,5 @@ func (r *KV) FullCompaction() error {
 		fk[i] = 0
 		lk[i] = 0xFF
 	}
-	return r.db.Compact(fk, lk)
```
Comment (coufalja) [inline internal/logdb/kv/pebble/kv_pebble.go:377]: The last (new) argument is if compaction should run in parallel, AFAIK the old behaviour was to not parallelise so I picked false here.

**Comment by coufalja** [comment]
@lni Sorry forgot about the Windows 🤦 https://github.com/lni/vfs/pull/2

**Comment by lni** [comment]
Thanks for the PR @coufalja

**Pull request merged**